File: Python/Minor_Assignment_4/Q13.py
L=eval(input("Enter list of tuples: "))
L.sort(key= lambda x : x[1])
print("After sorting: ",L)

'''
Output--
Enter list of tuples: [(1,5,7,4),(1,1,3),(4,2,1)] 
After sorting:  [(1, 1, 3), (4, 2, 1), (1, 5, 7, 4)]
'''

File: PPWC/Minor_Assignment_4/Q13.c
#include <stdio.h>
int main(void){
    int m = 10, n = 5;
    int *mp, *np;
    mp = &m;
    np = &n;
    *mp = *mp + *np;
    *np = *mp - *np;
    printf("%d %d\n%d %d\n", m, *mp, n, *np);
    return 0;
}

/*
Output--
15 15
10 10
*/